stacks/core/dev/stage-1.yml

```diff
@@ -50,6 +64,7 @@ services:
       - rabbitmq
       - users
       - things
+      - storage
```

Comment:
I think `babeltower` doesn't explicitly depends on the `storage` service. Maybe you thought it's dependent on why the later receives data from it, but all the process is asynchronous and anonymous.

Reply:
You're right! Removed it :smile: